Source/Dafny/Reporting.cs

```diff
@@ -195,7 +195,7 @@ public class ConsoleErrorReporter : BatchErrorReporter {
     public override bool Message(MessageSource source, ErrorLevel level, IToken tok, string msg) {
       if (base.Message(source, level, tok, msg) && ((DafnyOptions.O != null && DafnyOptions.O.PrintTooltips) || level != ErrorLevel.Info)) {
         // Extra indent added to make it easier to distinguish multiline error messages for clients that rely on the CLI
-        msg = msg.Replace(Environment.NewLine, Environment.NewLine + " ");
+        msg = msg.Replace("\n", "\n ");
```

Comment:
Nice find!

Reply:
Why was that needed?  I think the expectation here was that messages were going to use the native line ending, so it made sense to replace just that; is there an underlying problem?

Reply:
(Of course this is safe, but I wonder if it's hiding something else)

Reply:
The problem was that we are using native line endings almost nowhere, so this just didn't do anything on Windows, leading to inconsistent output between platforms.